source/internal/_xdropLastWhile.js

```diff
@@ -18,7 +19,7 @@ XDropLastWhile.prototype['@@transducer/step'] = function(result, input) {
                        : this.flush(result, input);
 };
 XDropLastWhile.prototype.flush = function(result, input) {
-  result = _reduce(
+  result = Z.reduce(
     this.xf['@@transducer/step'],
     result,
     this.retained
```

Comment:
I used `Z` functions wherever possible. They're simpler than the Ramda equivalents. They do not support transducing, they dispatch only to `fantasy-land/`-prefixed methods, and they don't contain special cases (vestigial or otherwise).

The sites at which I could *not* replace the application of a Ramda function with the application of its `Z` equivalent are where the dragons live. :dragon: